benchexec/tools/gazer-theta.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for gazer-theta
+    (combined tool of Gazer and Theta)
+    https://github.com/ftsrg/gazer
+    https://github.com/ftsrg/theta
+    """
+
+    REQUIRED_PATHS = ["gazer", "gazer/tools/gazer-bmc", "gazer/tools/gazer-theta"]
```

Comment:
Including `gazer` implies the rest. However, note that the default implementation of `program_files` treats `REQUIRED_PATHS` as relative to the executable's directory. So you would either need to change the content of the list or overwrite `program_files` (you can still call our utility method `_program_files_from_executable` with appropriate parameters if you wish).

Reply:
If I understand right what program_files does and if these paths are relative to the executable, would `"../../gazer"` (or simply `..`) suffice? *(Although even if it works, the executable will be included twice in the list.)* If not, I'll take a look at _program_files_from_executable and program_files and overwrite the latter.

Reply:
`".."` would effectively have the same effect as `"../../gazer"` but would be more robust in case of renamings of that directory. `test_tool_info` will tell you which things are included, so you can check with that.

If something is included twice (directly or indirectly) this is not a problem.